### ISSUE TITLE
Customize a Loss Function in Keras for a model of two outputs

### ISSUE THREAD

**Issue by rider-30** [rider-30/hello-world#2]
I have two outputs for a model: y_pred_1, and y_pred_2 and I want to design a loss function in Keras where I can use binary cross-entropy on both of them using individual loss weights. My idea of the loss function is like this:

    def my_loss_fn(y_true, y_pred_1, y_pred_2):

        loss_1 = tf.keras.losses.BinaryCrossentropy(y_true, y_pred_1)
        loss_2 = tf.keras.losses.BinaryCrossentropy(y_true, y_pred_2)
        loss = alpha*loss_1 + (1-alpha)*loss_2
        return loss

Here alpha is a predetermined value and the model will have to be trained based on this function. Is there a way to customize a loss function like this?
